LayoutTests/http/tests/security/cross-frame-access-get.html

```diff
@@ -7,10 +7,8 @@
             "Attr", 
             "Audio",
             "CDATASection", 
-            "CSSPrimitiveValue", 
             "CSSRule", 
             "CSSStyleDeclaration", 
-            "CSSValue", 
             "CharacterData", 
```

Comment:
I would restore these, and only remove these when the functionality has been removed.